lift_arm:
    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    
    # Position Controllers ---------------------------------------
    main_arm_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: main_arm_joint
      pid: {p: 1.0, i: 1.0, d: 0.0}
    upper_arm_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: upper_arm_joint
      pid: {p: 1.0, i: 1.0, d: 0.0}
    rack_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: rack_joint
      pid: {p: 1.0, i: 1.0, d: 0.0}
